### PR TITLE
Unicode support and py37 env

### PR DESCRIPTION
**Pull request by deti**
Adding support for Unicode and introducing tox env for Python 3.7

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -2,22 +2,27 @@
 envlist =
     py27-django{1.8.X,1.11.X}-djangocelery3.1.X-celery3.1.X,
     py27-django{1.8.X,1.11.X}-celery3.1.X,
```
Comment (bocribbz) [inline tox.ini]: > we can probably drop support for python 3.5, django 1.8 and celery 3

Comment (deti) [reply tox.ini]: I agree. Done!

**Comment by deti** [comment]
@bocribbz I believe travis would fail, because for successful test passing RabbitMQ is required. However, it's missing in our Travis.

Could I merge it into master myself?

**Comment by bocribbz** [comment]
> Could I merge it into master myself?

yup, i think we are good to merge into master.

**Pull request merged**